cranelift/zkasm_data/analyze-zkasm/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "analyze-zkasm"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = { workspace = true }
+clap = { workspace = true, default-features = true, features = ["std", "derive"] }
+cranelift-filetests = { workspace = true }
+wat = { workspace = true }
```

Comment:
Minor nit: Any reason to not this as a subcommand to `clif-tool`? If nothing else there would be fewer crates to link => faster build times.

https://github.com/near/wasmtime/tree/main/cranelift/src

Reply:
The motivation for adding a new crate was:

- This functionality is specific to the zkASM backend which is not part of upstream. So I added a new crate on a path that exists only on our fork. Besides separating concerns it might help to prevent conflicts when merging upstream.
  - On first glance (it might not be true), subcommands of `clif-util` seem to be backend agnostic. This would be in contrast to the functionality added here.
- Developing and reviewing the benchmarking infrastructure is more convenient if done incrementally. For instance one of the next steps will be adding a `run` subcommand to execute instrumented zkASM. At some point adding too many zkASM benchmarking specific commands to `clif-util` might be considered pollution?

I feel like there are arguments for both options and on my end there is no strong preference.